src/app/static/src/tests/components/workflowWizard/step.test.ts

```diff
@@ -17,7 +17,7 @@ describe(`step`, () => {
                 valid: false,
                 submitLabel: null
             },
-            slots: {default: runWorkflowReport}
+            slots: {default: "<div/>"}
```

Comment:
`runWorkflowReport` was causing some issues here because of the initialisation it now requires, but we shouldn't need to use the real component to test the step logic.